styles.css

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.logo-link {
+  color: #6070ff;
+  font-size: 25px;
+  font-weight: 700;
+}
+
+.bi-list {
+  margin-top: px;
+  width: 30px;
+  height: 30px;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+}
+
+.nav {
+  padding-left: 15px;
+  padding-right: 15px;
+  top: 0;
+  left: 0;
+  right: 0;
+  position: fixed;
+  z-index: 1000;
+}
+
+#intro {
+  background-image: url(./Assets/background-images/header-shapes\ mobile.png);
+  background-position: cente;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 100vh;
```

Comment:
- [ ] [OPTIONAL] kindly the page has a scrollbar after inspecting your CSS i realize it because of this `height` try to reduce it and let's make your code professional